test/jdk/javax/swing/JMenu/4213634/bug4213634.java

```diff
@@ -77,33 +71,30 @@ public  void createAndShowGUI() {
         frame.getContentPane().add("Center", ta);
         JButton button = new JButton("Test");
         frame.getContentPane().add("South", button);
-        frame.setBounds(100, 100, 400, 400);
+        frame.setSize(400, 400);
```

Comment:
Is this the best frame size? I see the text area has text to output, but the full sentence isn't even shown and there's no word wrap. Can change the text to anything else, especially since the TextArea text exceeds the line char limit. There's also a lot of empty space in this test frame. So, this can be changed in a variety of ways.